CMakeLists.txt

```diff
@@ -156,35 +120,14 @@ if(${BUILD_UNSTABLE_TOOLS})
 	add_subdirectory(Tools/GeneratorPerformanceTest/)
 endif()
 
-include(SetFlags.cmake)
-set_flags()
-set_lib_flags()
-enable_profile()
-
-# Under Windows, we need Lua as DLL; on *nix we need it linked statically:
-if (WIN32)
-	add_definitions(-DLUA_BUILD_AS_DLL)
-endif()
-
-# The Expat library is linked in statically, make the source files aware of that:
-add_definitions(-DXML_STATIC)
```

Comment:
Moved into submodule https://github.com/cuberite/expat/commit/968efc0b5479b167cec4a7aea8b84ed23adc10c3

Reply:
Why was this moved into a submodule, but the other submodules' configurations were left in? (`set(SQLITECPP_RUN_CPPLINT` et al.)

Reply:
It was moved since expat is our repository and there's no real upstream to update from. The others have maintained CMakeLists.texts so I think using their provided configuration options makes more sense.